doc/sphinx/particles.rst

```diff
@@ -436,79 +436,6 @@ For correct results, the LB thermostat has to be deactivated for virtual sites::
 
 
-..
-    .. _Virtual sites in the center of mass of a molecule:
-
-    Virtual sites in the center of mass of a molecule
-    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-    .. todo:: This is not implemented in Python, yet
-
-    To activate this implementation, enable the feature ``VIRTUAL_SITES_COM`` in :file:`myconfig.hpp`. Virtual sites are then placed in the center of mass of
-    a set of particles (as defined below). Their velocity will also be that
-    of the center of mass. Forces accumulating on the virtual sites are
-    distributed back to the particles which form the molecule. To place a
-    virtual site at the center of a molecule, perform the following steps in
-    that order
-
-    #. Create a particle of the desired type for each molecule. It should be
-       placed at least roughly in the center of the molecule to make sure,
-       its on the same node as the other particles forming the molecule, in
-       a simulation with more than one CPU.
-
-    #. Make it a virtual site using
-
-       part virtual 1
-
-    #. Declare the list of molecules and the particles they consist of:
-
-       analyze set { ...} ...
-
-       The lists of particles in a molecule comprise the non-virtual
-       particles as well as the virtual site. The id of this molecule is its
-       index in this list. For example,
-
-       analyze set {0 1 2 3 4} {0 5 6 7 8} {1 9 10 11}
-
-       declares three molecules, of which the first two consist of three
-       particles and a virtual site each (particles 14 and 58,
-       respectively). The third molecule has type 1 and consists of two
-       particles and a virtual site. The virtual sites were determined
-       before by setting the flag. You can choose freely one out of each
-       molecule, for example particles 1, 5, and 9.
-
-    #. Assign to all particles that belong to the same molecule the
-       molecules id
-
-       part mol
-
-       The molid is the index of the particle in the above list, so you
-       would assign 0 to particles 1-4, 1 to particles 5-8 and 2 to
-       particles 9-11. Alternatively, you can call
-
-       analyze set topo_part_sync
-
-       to set the s from the molecule declarations.
-
-    #. Update the position of all virtual particles (optional)
-
-       integrate 0
-
-    The type of the molecule you can choose freely, it is only used in
-    certain analysis functions, namely ``energy_kinetic_mol``,
-    ``pressure_mol`` and ``dipmom_mol``, which compute kinetic energy,
-    pressure and dipole moment per molecule type, respectively.
-
-    .. _Additional features:
-
-    Additional features
-    ~~~~~~~~~~~~~~~~~~~
-
-    The behavior of virtual sites can be fine-tuned with the following
-    switches in :file:`myconfig.hpp`.
-
-    - ``THERMOSTAT_IGNORE_NON_VIRTUAL`` specifies that the thermostat does not act on non-virtual particles
```

Comment:
this `THERMOSTAT_IGNORE_NON_VIRTUAL` macro is actually used, maybe we should keep it in the docs (revert deletion of L502-511), and un-indent it so that it shows up in the docs

Reply:
No, the usage should be remove too, it was just left-over in one place.